modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractGoCodegen.java

```diff
@@ -328,6 +329,16 @@ public String getSchemaType(Schema p) {
             type = openAPIType;
         return type;
     }
+    
+    @Override
+    public String toInstantiationType(Schema property) {
```

Comment:
Could you please provide a comment here on what this method does and why it does that? I *think* I understand it in the context of your PR, but if I was just looking at it in the codebase, I'd probably have no idea what's happening here and why.

Reply:
sure, will do.